.github/workflows/deploy-shutter-pages.yaml

```
@@ -0,0 +1,28 @@
+name: push shutter pages to webapp
+on:
+  workflow_call:
+    secrets:
+      WEBAPP_TOKEN:
+        required: true
+        description: Static webapp deployment token
+    inputs:
+      appname:
```

what about? it would be better than passing default as app name
```suggestion
     shutter_page_folder:
```